### PR TITLE
Feat/error operators

### PR DESCRIPTION
**Pull request by NelsonBN**
## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] Implemented tests for new features.
- [x] I tested the features already implemented.
- [x] Added usage examples in the readme.

### CODE REVIEW

**Review by NelsonBN — COMMENTED** [review]
All OK

**Review by NelsonBN — COMMENTED** [review]
Code smell fixed

**Review by filipe11402 — CHANGES_REQUESTED** [review]
Minor Questions made.
All Good aside from that

File: src/CommonUtils.cs [hunk 1]
```diff
@@ -4,10 +4,35 @@ namespace PowerUtils.Results
...
+            {
+                return false;
+            }
+
+            if(error.Property != other.Property)
+            {
+                return false;
+            }
+
+            if(error.Code != other.Code)
```
[the remark this answers is not in the document]
Comment (NelsonBN) [reply src/CommonUtils.cs]: I agree with you. Updated it

File: src/CommonUtils.cs [hunk 2]
```diff
@@ -4,10 +4,35 @@ namespace PowerUtils.Results
...
+            {
+                return false;
+            }
+
+            if(error.Description != other.Description)
```
[the remark this answers is not in the document]
Comment (NelsonBN) [reply src/CommonUtils.cs]: I agree with you. Updated it

**Review by filipe11402 — APPROVED** [review]
All Ok

**Pull request merged**